lms/envs/common.py

```diff
@@ -2894,6 +2894,14 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     #   For more information, check https://github.com/openedx/edx-platform/pull/13388 and
     #   https://github.com/openedx/edx-platform/pull/14571.
     TASK_MAX_RETRIES=5,
+
+    #block structures are stored in a more permanent storage
+    #which provides an additional backup for cache misses
+    STORAGE_CLASS='django.core.files.storage.FileSystemStorage',
+    STORAGE_KWARGS=dict(
+        location=MEDIA_ROOT,
+        base_url=MEDIA_URL,
+    ),
```

Comment:
Hmm, what is the error message that you see when you don't include this? I think this backend is already configured in a different place: https://github.com/openedx/edx-platform/blob/master/lms/envs/common.py#L1133-L1144

Reply:
@dianakhuang OK, I will remove it. Thanks